packages/website/package.json

```diff
@@ -3,6 +3,9 @@
 	"version": "0.1.0",
 	"sideEffects": false,
 	"scripts": {
+		"init-db": "prisma init --datasource-provider sqlite",
```

Comment:
That won't work. There is already a directory called `prisma`. You need `prisma generate` instead.

BTW, please provide `"init-db": "run-s init-db:*",` too to run three scripts at once.